functions/index.js

```diff
@@ -60,6 +60,53 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.createNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var title = request.body.data.Title;
+        var des = request.body.data.Description;
+        var startDate = request.body.data.StartDate;
+        var endDate = request.body.data.EndDate;
```

Comment:
endDate will also use the same method to get the date and must not get it from ui.